certif/tests/acceptance/session-add-candidate_test.js

```diff
@@ -0,0 +1,69 @@
+import { visit as visitScreen } from '@1024pix/ember-testing-library';
+import { click, fillIn } from '@ember/test-helpers';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+import { setupApplicationTest } from 'ember-qunit';
+import { module, test } from 'qunit';
+
+import { authenticateSession } from '../helpers/test-init';
+
+module('Acceptance | Session Add Candidate', function (hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+
+  module('When certification center is V3 pilot for pix/pix+ split', function () {
```

Comment:
```suggestion
  module('When certification center has V3 and complementary alone pilots', function () {
```

Reply:
@AndreiaPena "is"

Reply:
ouais ouais mais en vrai c'est un peu flou. Un centre EST V3 / ASSOCIÉ à une fonctionnalité pilote V3 / ABONNÉ à ....
Bref je me perd dans tout ça

Reply:
Allez, moi aussi j'en propose une :D
'When certification center is pix/pix+ split pilot and V3 pilot"